OrbitGl/App.cpp

```diff
@@ -123,7 +124,7 @@ void OrbitApp::OnKeyAndString(uint64_t key, std::string str) {
 }
 
 void OrbitApp::OnCallstack(CallStack callstack) {
-  Capture::GSamplingProfiler->AddUniqueCallStack(callstack);
+  Capture::capture_data_.GetCallstackData()->AddUniqueCallStack(callstack);
```

Comment:
I am thinking about if it would be maybe better to hide those methods to CaptureData, and then call CaptureData->AddUniqueCallstack. What do you think? Could be also in a separate PR, and I am not quite sure about this.

Reply:
Added these methods to CaptureData.